algorithms/appo/learner.py

```diff
@@ -399,6 +412,26 @@ def _value_loss(self, new_values, old_values, target, clip_value):
 
         return value_loss
 
+    def entropy_exploration_loss(self, action_distribution):
+        entropy = action_distribution.entropy()
+        if self.cfg.exploration_loss_coeff > 0.0:
```

Comment:
These ifs are a bit redundant now, because you already check exploration_loss_coeff in the beginning

Reply:
Done